app/vep/utils/vcf_results.py

```diff
@@ -1,10 +1,11 @@
 """ Module for loading a VCF and parsing it into a VepResultsResponse
 object as defined in APISpecification"""
 
+from typing import List, Dict, Any, IO
+
 import vcfpy
-from typing import List, Dict, Any
 
-from vep.models import vcf_results_model as model
+from app.vep.models import vcf_results_model as model
```

Comment:
do you need `app.`? complained for me locally.

Reply:
Aah! yes to run the test you need to access it using `app.`  but not for the app.

Reply:
Yeah I needed that and IO before the unit tests would run